README.md

```diff
@@ -97,7 +97,7 @@ Setup for developers (Windows)
     $ CREATE USER <anyname you want e.g systers> WITH PASSWORD 'your password';
     $ CREATE DATABASE systersdb;
     $ \c systersdb;
-    $ GRANT ALL PRIVILEGES ON systersdb TO <username created above>;
+    $ GRANT ALL PRIVILEGES ON DATABASE systersdb TO <username created above>;
```

Comment:
I think, considering to the previous context, it is automatically referring to a Database, need not mention explicitly.

Reply:
The old command does not work on Windows. I have tried it.